server/src-lib/Hasura/RQL/DDL/Permission.hs

```diff
@@ -258,6 +265,7 @@ instance IsPerm SelPerm where
 data UpdPerm
   = UpdPerm
   { ucColumns :: !PermColSpec -- Allowed columns
+  , ucSet     :: !(Maybe Object) -- Preset columns
```

Comment:
Why not `(Maybe (M.HashMap PGCol Value)`? You then can just use `forM` in `procSetObj`